### PR TITLE
Added feedback table

### PR DESCRIPTION
**Pull request by tej99**
Assumptions made:
- 'survey_id' will never be more than 25 characters long 
- 'period' is year/month which will not be more than 25 characters long

### CODE REVIEW

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
Can you update the CHANGELOG? Also, are 25 chars guaranteed to always be enough?

**Review by warrenbailey — DISMISSED** [review]

**Comment by warrenbailey** [comment]
Good point @JamesGardiner I missed that, the SQLAlchemy doesn't restrict the size of the String so I think it's possible that it could exceed 25 chars. (https://github.com/ONSdigital/sdx-store/pull/62/files)

File: postgres/schema.sql [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CREATE TABLE IF NOT EXISTS responses (tx_id uuid PRIMARY KEY, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb);
+CREATE TABLE IF NOT EXISTS feedback_responses (id SERIAL, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb, survey char(25), period char(25));
```
Comment (warrenbailey) [inline postgres/schema.sql]: What's the reason for 25 chars?

Comment (tej99) [reply postgres/schema.sql]: I'm not sure if you can create a column with data type char without having a size and 25 chars seemed sensible. I can change the SQLAlchemy to restrict the size to 25? Unless there's a better way to do this?

Comment (JamesGardiner) [reply postgres/schema.sql]: Use an `@property` and validate the string length in an `@property.setter` method, raising an exception if necessary

Comment (tej99) [reply postgres/schema.sql]: SQLAlchemy handles that

**Review by JamesGardiner — DISMISSED** [review]

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**